src/lib/Renderer.js

```diff
@@ -950,15 +950,15 @@ var Renderer = Base.extend('Renderer', {
 
         var grid = this.grid,
             behavior = grid.behavior,
-            baseProperties = behavior.getColumnProperties(c);
+            column = behavior.getActiveColumn(c),
+            cellProperties = column && column.getCellProperties(r),
+            baseProperties = cellProperties || column && column.getProperties();
```

Comment:
So here we get the combined (cascaded) cell props object _when it exists_ and only when it doesn't do we fall back to the column props obj.